packages/plugin-e2e/src/e2e-selectors/types.ts

```diff
@@ -196,9 +196,23 @@ export type Pages = {
             generalTypeSelectV2: string;
             previewOfValuesOption: string;
             submitButton: string;
+            generalNameInputV2: string;
+            applyButton: string;
           };
         };
       };
+      Close: string;
```

Comment:
Looks like this button already [exist](https://github.com/grafana/grafana/blob/main/packages/grafana-e2e-selectors/src/selectors/pages.ts/#L94) in grafana/e2e-selectors, but on a different level in the structure. Would be good if these align as we plan to merge these selectors with e2e-selectors soon.

Reply:
Have pushed changes to address this, please lmk if I've moved this to the correct place